Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/CommonSettings.java

```diff
@@ -152,22 +168,49 @@ private static void setEnvironment(@NotNull String env, Map<String, String> endP
     }
 
     private static boolean isUsingDeprecatedBaseFolder(String baseDir, String deprecated) {
-        return !FileUtil.isNonEmptyFolder(baseDir) && FileUtil.isNonEmptyFolder(deprecated);
+        File baseFile = new File(baseDir);
+        return !baseFile.exists() && FileUtil.isNonEmptyFolder(deprecated);
     }
 
     private static void initBaseDir(@NotNull String basePath) throws IOException {
         File baseDir = new File(basePath);
         if(!baseDir.exists()){
             FileUtils.forceMkdir(baseDir);
         }
-        Files.setAttribute(baseDir.toPath(), "dos:hidden", true);
+        if (Utils.isWindows()) {
+            Files.setAttribute(baseDir.toPath(), "dos:hidden", true);
+        }
+    }
+
+    private static void moveResourcesToBaseFolder(String baseDir, String deprecatedDir) {
+        final File baseFile = new File(baseDir);
+        final File deprecatedFile = new File(deprecatedDir);
```

Comment:
bad name of deprecatedFile, use  deprecatedDir

Reply:
Renamed